--- charts-original/templates/_helpers.tpl
+++ charts/templates/_helpers.tpl
@@ -129,3 +129,11 @@
 {{ (split "@" (default $.Chart.AppVersion $.Values.image.tag))._0 }}
 {{- end -}}
 
+
+{{- define "system_default_registry" -}}
+{{- if .Values.global.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.systemDefaultRegistry -}}
+{{- else -}}
+{{- "" -}}
+{{- end -}}
+{{- end -}}
\ No newline at end of file
